src/crewai/types/crew_types.py

```diff
@@ -0,0 +1,61 @@
+from typing import (
```

Comment:
I didn't quite get why you added it or where it’s being used. Did you maybe forget to include some code in this PR?